src/java/org/apache/cassandra/db/Directories.java

```diff
@@ -492,18 +492,39 @@ public boolean hasAvailableDiskSpace(long estimatedSSTables, long expectedTotalW
     {
         long writeSize = expectedTotalWriteSize / estimatedSSTables;
         long totalAvailable = 0L;
+        boolean hasSpace = true;
```

Comment:
where is this used?

Reply:
Nowhere, apparently. It is used in the trunk version, where this check has a different structure.

I'll remove it and thanks.